client/src/components/Post/PostLikePanel/PostLikePanel.jsx

```diff
@@ -45,15 +43,17 @@ const PostLikePanel = ({ post, user }) => {
         <IconButton aria-label="comments">
           <ChatBubbleOutlineIcon />
         </IconButton>
-        {post.comments.length}
+        {comments.length}
       </div>
     </Fragment>
   )
 }
 
 PostLikePanel.propTypes = {
+  id: PropTypes.number,
```

Comment:
I guess, it should be PropTypes.number.isRequired, as well as user likes and comments (they'll be empty arrays, but still)